tests/UI/campaigns/functional/BO/11_international/03_taxes/taxRules/03_bulkActions.js

```diff
@@ -0,0 +1,185 @@
+    await expect(numberOfTaxRules).to.be.above(0);
+  });
+
+  // 1 : Create 2 tax rules with data from faker
+  describe('Create 2 Tax rules in BO', async () => {
+    const tests = [
+      {args: {taxRuleToCreate: firstTaxRuleData}},
+      {args: {taxRuleToCreate: secondTaxRuleData}},
+    ];
+
+    tests.forEach((test, index) => {
+      it('should go to add new tax rule page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `goToNewTaxRulePage${index + 1}`, baseContext);
+
+        await taxRulesPage.goToAddNewTaxRulesGroupPage(page);
+
+        const pageTitle = await addTaxRulesPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(addTaxRulesPage.pageTitleCreate);
+      });
+
+      it('should create tax rule and check result', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `CreateTax${index + 1}`, baseContext);
+
+        const textResult = await addTaxRulesPage.createEditTaxRulesGroup(page, test.args.taxRuleToCreate);
+        await expect(textResult).to.contains(addTaxRulesPage.successfulCreationMessage);
+      });
+
+      it('should go to Tax Rules page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `goToTaxRulesPage${index}`, baseContext);
+
+        await taxesPage.goToTaxRulesPage(page);
+
+        const numberOfLineAfterCreation = await taxRulesPage.getNumberOfElementInGrid(page);
+        await expect(numberOfLineAfterCreation).to.be.equal(numberOfTaxRules + index + 1);
+
+        const pageTitle = await taxRulesPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(taxRulesPage.pageTitle);
+      });
+    });
+  });
+
+  // 2 : Enable/Disable with bulk actions
+  describe('Enable and Disable Tax rules with Bulk Actions', async () => {
+    const tests = [
+      {args: {taxRule: firstTaxRuleData.name, action: 'disable', enabledValue: false}, expected: 'Disabled'},
+      {args: {taxRule: secondTaxRuleData.name, action: 'enable', enabledValue: true}, expected: 'Enabled'},
+    ];
+
+    tests.forEach((test, index) => {
+      it('should filter list by name', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `filterTaxesToChangeStatus${index}`, baseContext);
+
+        await taxRulesPage.filterTable(
+          page,
+          'input',
+          'name',
+          test.args.taxRule,
+        );
+
+        const textResult = await taxRulesPage.getTextColumnFromTable(page, 1, 'name');
+        await expect(textResult).to.contains(test.args.taxRule);
+      });
+
+      it(`should ${test.args.action} tax rules with bulk actions and check Result`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `bulk${test.args.action}`, baseContext);
+
+        const textResult = await taxRulesPage.changeEnabledColumnBulkActions(page, test.args.enabledValue);
+
+        /* Successful message is not visible, skipping it */
```

Comment:
Is this a known issue ? We need to keep track of this to be sure to re-enable it when it's fixed

Reply:
Okey, I will check if there is an issue :+1: